### ISSUE TITLE
Add term hemimelia of lower leg

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10485]
**Preferred term label:**
Hemimelia of lower leg

**Synonyms**

**Definition (free text, please give PubMed ID)**
A developmental defect leading to a transverse deficiency with absence of the lower leg (eg tibia and fibula), ankle and foot. 
PMID: 26172360, ORPHA:294981

**Parent term (use hpo.jax.org/app)**
Transverse terminal lower limb defect (#10480)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Adams-Oliver syndrome (OMIM100300)

**Your nano-attribution (ORCID)**
A. Zankl 0000-0001-8612-1062, @azankl
L. Wit 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl
Adding new term: 
- ID: HP:6000947
- label: Hemimelia of lower leg
- parents: 
- pmid: PMID:26172360
- def: A developmental defect leading to a transverse deficiency with absence of the lower leg (eg tibia and fibula), ankle and foot.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10485
- synonyms:

**Comment by pnrobinson** [comment]
Added to parent  Transverse terminal lower limb defect
